### PR TITLE
Add Table Slice to BlogPost

### PR DESCRIPTION
**Pull request by vuphuctho**
- Create new slice 'Table Slice' with variant '2 Columns with Icon'
- Add Table Slice to BlogPost
- Move rendering logic of old BlogLeagueTable component into new slice component for Table Slice
- Remove logic related to fetching and displaying data from Google Sheet

I would need 2-3 days to populate data from Google Sheet to Prismic, but the core implementation should be ready for review

### CODE REVIEW

File: slices/TableSlice/index.vue
Comment (sopoour) [inline slices/TableSlice/index.vue:2]: not sure about those width calculations. What do you try to achieve with that? I'd just keep it full so it's in the same width as all the rest of the content

**Review by sopoour — APPROVED** [review]
This looks really good!! Just small comments :) **Before we merge this we need to add the remaining brands to Prismic**. DON'T MERGE until that's done 🙏

File: slices/TableSlice/index.vue
Comment (vuphuctho) [reply slices/TableSlice/index.vue:2]: @sopoour since the team is fine with the current design, I added the styling to maintain same width. It is not important though; we can remove the styling.

File: slices/TableSlice/index.vue
Comment (vuphuctho) [reply slices/TableSlice/index.vue:22]: @sopoour ok, I will add padding to column's content if there is no icon, to align the text content.

Comment (vuphuctho) [reply slices/TableSlice/index.vue]: @sopoour fixed in latest commits

File: slices/TableSlice/index.vue
Comment (vuphuctho) [reply slices/TableSlice/index.vue]: @sopoour styling removed in latest commits

**Comment by vuphuctho** [comment]
@sopoour @RogerTangos all banks in the original Google Sheet are added into Prismic

**Pull request merged**
